### DISCUSSION TITLE
docker buildx build error "The directory name is invalid."

### DISCUSSION THREAD

**Discussion by mkim0203** [docker/buildx#1808] [Q&A]
1. create 
   docker buildx create --use --driver=docker-container --name=hc-bulid --config /https/buildkit-hc-build.toml
2. build
   docker buildx build -t test.com:30008/temp/backendapi:0.0.1 -f C:\src\Dockerfile.gen --cache-to type=local,dest=/data/docker-cache --cache-from type=local,src=/data/docker-cache C:\src --push --provenance=false

[+] Building 2.2s (1/1) FINISHED
 **=> ERROR [internal] booting buildkit**                                                                                                                                                       
 => => pulling image moby/buildkit:buildx-stable-1
 => => creating container buildx_buildkit_hc-bulid0
ERROR: mkdir C:\Users\mhkim\AppData\Local\Temp\buildkitd-config1566081107\etc\buildkit\certs\test.com:30008: The directory name is invalid.

* buildx version : github.com/docker/buildx v0.10.4 c513d34049e499c53468deac6c4267ee72948f02
* os : windows 11
* config file 
## buildkit-hc-build.toml
[registry."test.com:30008"]
  ca=["/https/host-ssl.crt"]

이것은 버그 인가요? 윈도우 환경에서 디렉토리 이름에 ":"가 들어갈수 없어서 발생되는 현상으로 보입니다. ubuntu 에서 같은 방법으로 진행시 에러가 발생하지 않습니다.
========
is this a bug? It seems to be a phenomenon that ":" cannot be entered in the directory name in the Windows environment. When proceeding in the same way in ubuntu, no error occurs.

**Comment by crazy-max** [comment]
It seems to be a bug yes because your registry specified a port. Will take a look at this one.
